tests/attr/test_feature_ablation.py

```diff
@@ -345,15 +354,28 @@ def _multi_input_batch_scalar_ablation_assert(self, func):
 
     def _ablation_test_assert(
         self,
-        model,
-        test_input,
-        expected_ablation,
-        feature_mask=None,
-        additional_input=None,
-        ablations_per_eval=(1,),
-        baselines=None,
-        target=0,
-    ):
+        model: Union[Callable[[Any], Any]],
+        # test_input: Union[TensorOrTupleOfTensors, Tuple[Union[str, Tensor], Tensor]],
+        test_input: TensorOrTupleOfTensors,
+        expected_ablation: Union[
+            List[List[float]],
+            Tuple[List[List[float]], List[List[Any]]],
+            Tuple[Tensor, Tensor],
+            List[float],
+            Tuple[List[List[float]]],
+            Tuple[List[List[float]], List[List[float]]],
+            Tuple[List[List[float]], List[List[float]], List[List[float]]],
+        ],
```

Comment:
This type hint can be simplified to just Union[
            List[float],
            List[List[float]],
            Tuple[List[List[float]], ...],
            Tuple[Tensor, ...],
]

Note that this will cause an issue with an empty list, so for line 215, you would need to add
`exp: Tuple[List[List[float]], ...] = ([[9.0, -3.0, 12.0]], [[]],)` and pass exp to _ablation_test_assert.